### PR TITLE
feat (api): handlers error management

### PR DESCRIPTION
**Pull request by fsamin**

### CODE REVIEW

File: engine/api/environment_group.go [hunk 1]
```diff
@@ -27,58 +27,50 @@ func updateGroupRoleOnEnvironmentHandler(w http.ResponseWriter, r *http.Request,
 	data, err := ioutil.ReadAll(r.Body)
 	if err != nil {
 		log.Warning("updateGroupRoleOnEnvironmentHandler: Cannot read body :%s", err)
-		WriteError(w, r, sdk.ErrUnknownError)
-		return
+		return sdk.ErrUnknownError
```
Comment (sguiheux) [inline engine/api/environment_group.go]: ErrWrongRequest

File: engine/api/environment_group.go [hunk 2]
```diff
@@ -88,57 +80,49 @@ func addGroupInEnvironmentHandler(w http.ResponseWriter, r *http.Request, db *go
...
-		return
+		return sdk.ErrWrongRequest
 	}
 
 	env, err := environment.LoadEnvironmentByName(db, key, envName)
 	if err != nil {
 		log.Warning("addGroupInEnvironmentHandler: Cannot load %s: %s\n", envName, err)
-		w.WriteHeader(http.StatusNotFound)
-		return
+		return sdk.ErrNotFound
```
Comment (sguiheux) [inline engine/api/environment_group.go]: return err

File: engine/api/environment_group.go [hunk 3]
```diff
@@ -88,57 +80,49 @@ func addGroupInEnvironmentHandler(w http.ResponseWriter, r *http.Request, db *go
...
 	}
 
 	g, err := group.LoadGroup(db, groupPermission.Group.Name)
 	if err != nil {
 		log.Warning("addGroupInEnvironmentHandler: Cannot find %s: %s\n", groupPermission.Group.Name, err)
-		w.WriteHeader(http.StatusNotFound)
-		return
+		return sdk.ErrNotFound
```
Comment (sguiheux) [inline engine/api/environment_group.go]: return err

**Comment by fsamin** [comment]
Has been merged into https://github.com/ovh/cds/pull/195

**Pull request closed without merging**